pkg/api/node.go

```diff
@@ -33,46 +33,45 @@ import (
 	"k8s.io/klog"
 	"k8s.io/metrics/pkg/apis/metrics"
 	_ "k8s.io/metrics/pkg/apis/metrics/install"
-	"sigs.k8s.io/metrics-server/pkg/provider"
 )
 
-type MetricStorage struct {
+type NodeMetricsAPI struct {
```

Comment:
The package is called `api` and the structs are also suffixed with `API` at the same time, hence this stutters. I suggest to remove the suffix.

question: I suppose those structs are not used anywhere else, i.e. we don't have to maintain an external API? This "api" is just internal to metrics-server?

Reply:
It's internal, renamed to nodeMetrics as suggested